Tasks/Koval/Task_2/Task2.py

```diff
@@ -0,0 +1,161 @@
+# Реализовать:
+# 1.текстовый вывод текущего времени
+# 2.текстовый вывод времени, введённого с консоли (hh:mm).
+# (дать пользователю возможность выбрать режим работы программы)
+
+# Для получения текущего времени использовать модуль datetime.
+
+# min == 0: такое-то значение часа ровно
+# min < 30: столько-то минут следующего часа
+# min == 30: половина такого-то
+# min > 30 and min < 45 столько-то минут следующего часа
+# min >= 45 без min такого-то
+
+
+# 21:16
+# шестнадцать минут десятого 
+
+import datetime
+
+hour_in_true_form = {
+    0 : 'ноль',
+    1 : 'один',
+    2 : 'два',
+    3 : 'три',
+    4 : 'четыре',
+    5 : 'пять',
+    6 : 'шесть',
+    7 : 'семь',
+    8 : 'восемь',
+    9 : 'девять',
+    10 : 'десять',
+    11 : 'одиннадцать',
+    12 : 'двеннадцать',
+    13 : 'тринадцать',
+    14 : 'четырнадцать',
+    15 : 'пятнадцать',
+    16 : 'шестнадцать',
+    17 : 'семнадцать',
+    18 : 'восемнадцать',
+    19 : 'девятнадцать',
+    20 : 'двадцать',
+    21 : 'двадцать один',
+    22 : 'двадцать два',
+    23 : 'двадцать три',
+    100: 'час'
+}
+
+half_hour_helper = {
+    1 : 'первого',
+    2 : 'второго',
+    3 : 'третьего',
+    4 : 'четвертого',
+    5 : 'пятого',
+    6 : 'шестого',
+    7 : 'седьмого',
+    8 : 'восьмого',
+    9 : 'девятого',
+    10 : 'десятого',
+    11 : 'одиннадцотого',
+    12 : 'двеннадцатого'
+}
+
+min_in_true_form = {
+    1 : 'одна',
+    2 : 'две',
+    3 : 'три',
+    4 : 'четыре',
+    5 : 'пять',
+    6 : 'шесть',
+    7 : 'семь',
+    8 : 'восемь',
+    9 : 'девять',
+    10 : 'десять',
+    11 : 'одиннадцать',
+    12 : 'двенадцать',
+    13 : 'тринадцать',
+    14 : 'четырнадцать',
+    15 : 'пятнадцать',
+    16 : 'шестнадцать',
+    17 : 'семнадцать',
+    18 : 'восемнадцать',
+    19 : 'девятнадцать',
+    20 : 'двадцать',
+    21 : 'двадцать одна',
+    22 : 'двадцать две',
+    23 : 'двадцать три',
+    24 : 'двадцать четыре',
+    25 : 'двадцать пять',
+    26 : 'двадцать шесть',
+    27 : 'двадцать семь',
+    28 : 'двадцать восемь',
+    29 : 'двадцать девять',
+    31 : 'тридцать одна',
+    32 : 'тридцать две',
+    33 : 'тридцать три',
+    34 : 'тридцать четыре',
+    35 : 'тридцать пять',
+    36 : 'тридцать шесть',
+    37 : 'тридцать семь',
+    38 : 'тридцать восемь',
+    39 : 'тридцать девять',
+    40 : 'сорок',
+    41 : 'сорок одна',
+    42 : 'сорок две',
+    43 : 'сорок три',
+    44 : 'сорок четыре'
+}
+
+min_in_without_form = {
+    45 : 'пятнадцати',
+    46 : 'четырнадцати',
+    47 : 'тринадцати',
+    48 : 'двеннадцати',
+    49 : 'одиннадцати',
+    50 : 'десяти',
+    51 : 'девяти',
+    52 : 'восьми',
+    53 : 'семи',
+    54 : 'шести',
+    55 : 'пяти',
+    56 : 'четырёх',
+    57 : 'трёх',
+    58 : 'двух',
+    59 : 'одной',
+}
```

Comment:
it's a very through data structure, but it has some overlapping values, it can be simplified to a three or even two dictionaries with a similar structure